data/styles/default/include/Basemap_label.mapcss

```diff
@@ -362,16 +364,33 @@ node|z12[place=suburb]
 {font-size: 11;}
 node|z12[place=suburb]::int_name
 {font-size: 9;}
-node|z13-14[place=suburb]
+node|z13[place=suburb]
 {font-size: 12;}
-node|z13-14[place=suburb]::int_name
+node|z13[place=suburb]::int_name
 {font-size: 10;}
+node|z14[place=suburb]
+{font-size: 14;}
+node|z14[place=suburb]::int_name
+{font-size: 12;}
+
+node|z13[place=quarter],
+{font-size: 10;}
+node|z13[place=quarter]::int_name,
+{font-size: 9;}
+node|z14[place=quarter],
+{font-size: 12;}
+node|z14[place=quarter]::int_name,
+{font-size: 10;}
+node|z15-[place=quarter],
+{font-size: 14;}
+node|z15-[place=quarter]::int_name,
+{font-size: 12;}
 
 node|z13-14[place=locality],
-node|z13-14[place=neighbourhood],
+node|z14[place=neighbourhood],
```

Comment:
Actually I'm concerned that making neighbourhoods one zoom level less visible will be detrimental for many countries that don't use quarters.

E.g. I've checked Romania, Russia, Bulgaria - most cities don't map quarters, but suburbs + neighbourhoods.
(also many mapped place=quarter features are actually place=city_block due to the same word used for them in some eastern european languages)

So it'd be better to put this change through an alpha version first and gather users' feedback.
Or retain neighbourhoods at z13- and differentiate them with quarters by font size only.

Reply:
Ok, I will change the PR to render `neighbourhood` from z13

Reply:
Thanks! Then it'd make sense to retain the type id for `place|neighbourhood;962;` and make place-quarter a new entry `place|quarter;907;`. This way the rendering won't change at all for old maps.